### ISSUE TITLE
underused solar panels

### ISSUE THREAD

**Issue by downRightCoder256** [endless-sky/endless-sky#4885]
**Is your feature request related to a problem? Please describe.**
at current point in game, there are two human outfits: KP-6 Photovoltaic panel and KP-6 Photovoltaic array, that seem to be more or less obsolete. Main issue for that would be terrible energy generation per ton and energy generation per credit features
**Related Issue Links**
     #4065
     #4034

**Describe the solution you'd like**
-make solar panels a bit better:
     make solar panels produce around 50% more energy. so far probably the simplest solution
-have more non combat ships equipped with it by default
     have some ships that aren't designed for direct combat utilize such panels.good examples
     i would like to point out are star barge and space shuttle: the only real big energy consumer
     on 'barge is AM turret. having it equipped with 2 solar panels right off the bat would
     greatly reduce its purchase cost, and make for a more energy managing experience
-make them more available
     knowing that they have been around for a while, and that their original company is now
     defunct, it would be expected to find such an outfit on almost any station with an outfitter,
     and since it can be easily manufactured out of cheap components (some plastic, silicium,
     and some conductor). it would be expected for such a piece of electronics to be dirt cheap;
     around 2000 credits on average and probably no more that 5000 credits in worst case
     scenario
-make "subships"(drones & fighters) able to recharge
     as of 0.9.11, fighters and drones can be repaired by its mothership, but their batteries cannot
     be recharged. Adding such an ability would greatly increase the demand for solar panels,
     since they are cheaper than fuel cells (and could be cheaper if points above are considered),
     and most fighters and drones aren't designed to last long enough in combat to even deplete
     a LP036a battery, the smallest of them all, such cheap power source would probably fix the
     issue of demand for "subships".

edit: since I've only now noticed it may not make sense at first; each of these points is its own proposal, not a single unified solution.

**Comment by downRightCoder256** [comment]
my proposal of the solution on #4034 problem is to make fighters either dirt cheap (<200,000 credits per unit), or more combat effective. last point tackles the former

**Comment by Zitchas** [comment]
My PR doesn't affect the usage of solar panels, but it does give them a bit of a boost in that they now act much more like their description states: Things that use only minimal space in the ship and are bolted on the outside - but still provide a fairly hefty mass, thus impacting mobility. 

It is a rather minor change, but it could be enough to make them more desirable for at least poor merchants that are trying to maximize cargo space and don't need to power weaponry.

**Comment by downRightCoder256** [comment]
maybe we could consider the availbility point now? it seems lore friendly

**Comment by Amazinite** [comment]
I've merged #4886. Do you think there's anything else we should do for solar panels? If not, we should close this issue.

**Comment by downRightCoder256** [comment]
maybe make some ships, especially freighters and transport ships be equipped with it by default and/or make them a bit cheaper. but other that that, this is ready to be closed

**Comment by downRightCoder256** [comment]
I am specifically looking at a star barge here, but if you consider it a frivilous enough to leave it as be, you can close the issue as-is.

**Comment by Amazinite** [comment]
> maybe make some ships, especially freighters and transport ships be equipped with it by default

I'll tackle this later; I have the perfect idea for what to do.

> I am specifically looking at a star barge here,

I think I want to avoid putting solar panels on the starter ships. The game already throws enough at new players, no need to have a certain start also have more complex energy generation mechanics.

**Comment by downRightCoder256** [comment]
maybe we should make fighters and drones be able to recharge off of motherships generators once docked, enabling players to effectivelly use solar panels on them too. on a boxwing, if player replaces FC with a solar panel and a supercap, it can be installed with an cargo expansion; neat for scrappy players

**Comment by downRightCoder256** [comment]
opened a new issue to discuss this problem separately

**Comment by downRightCoder256** [comment]
> I have the perfect idea for what to do.
can we hear ?

**Comment by Amazinite** [comment]
763f5d9
Reduced the cost of solar panels and gave them to a number of Tarazed and Lionheart ships. My "perfect idea" was to give the Star Queen only solar panels as its energy generation, which I also did in that commit. 
Note that I did increase the space of the solar panels from what they were in #4886 to a middle ground. I think that PR went a little overboard; it is important to note that solar collection doubles when very close to the system center, and the change from #4886 made it so that solar panels were basically on par with the stronger reactors in terms of energy/space, even before taking that doubling into account.
